packages/semver/src/builders/version/builder.spec.ts

```diff
@@ -141,6 +155,84 @@ describe('@jscutlery/semver:version', () => {
     expect(output).toEqual(expect.objectContaining({ success: false }));
   });
 
+  describe('Plugins', () => {
+    /* eslint-disable @typescript-eslint/no-var-requires */
+    const { publish: npmPublish } = require('@custom-plugin/npm');
+    const { publish: githubPublish } = require('@custom-plugin/github');
+    const {
+      publish: semanticPublish,
+      addChannel: semanticAddChannel,
+    } = require('@semantic-release/npm');
+    /* eslint-enable @typescript-eslint/no-var-requires */
+
+    beforeEach(() => {
+      npmPublish.mockResolvedValue('');
+      githubPublish.mockResolvedValue('');
+      semanticPublish.mockResolvedValue('');
+      semanticAddChannel.mockResolvedValue('');
+    });
+
+    afterEach(() => {
+      npmPublish.mockRestore();
+      githubPublish.mockRestore();
+      semanticPublish.mockRestore();
+      semanticAddChannel.mockRestore();
+    });
+
+    it('should run publish hook', async () => {
+      const output = await runBuilder(
+        {
+          ...options,
+          plugins: ['@custom-plugin/npm', '@custom-plugin/github'],
+        },
+        context
+      ).toPromise();
+
+      expect(npmPublish).toBeCalledTimes(1);
+      expect(githubPublish).toBeCalledTimes(1);
+      expect(
+        semanticPublish /* @semantic-release/npm not declared and not called. */
+      ).not.toBeCalled();
+      expect(output).toEqual(expect.objectContaining({ success: true }));
+    });
+
+    it('should handle plugin configuration', async () => {
+      const output = await runBuilder(
+        {
+          ...options,
+          plugins: [
+            '@custom-plugin/npm',
+            ['@custom-plugin/github', { remoteUrl: 'remote' }],
```

Comment:
As mentioned in `plugin.ts`, this could be:
```suggestion
            '@custom-plugin/npm',
            { module: '.../github', options: {remoteUrl: 'remote'}},
```